pkg/client/unit.go

```diff
@@ -76,10 +93,10 @@ func (u *Unit) Type() UnitType {
 }
 
 // Expected returns the expected state and config for the unit.
-func (u *Unit) Expected() (UnitState, string) {
+func (u *Unit) Expected() (UnitState, UnitLogLevel, *proto.UnitExpectedConfig) {
```

Comment:
Is there a reason why the log level is being sent as a separate return value?

Reply:
This is moved to top-level because it will allow the Elastic Agent to change a units log level without modifying the configuration that comes from the policy.

The idea here is that with Elastic Agent running in any mode, you can use the Elastic Agent CLI to call into the running daemon on change a unit to say trace. That will allow debug-ability of that single unit, without having to modify the policy.

I still think that Elastic Agent can read `log_level` from the configuration as the default `log_level` for a unit, but that will come here in a top-level and not shown in the configuration, just like `use_output`. Then for debugging purposes that `log_level` can be changed different from what is in the policy.